hls_vi_historical/main.py

```diff
@@ -94,7 +113,12 @@ def make_s3_downloader(s3: S3Client) -> Downloader:
     def s3_download_file(src: tuple[str, str], dst: Path) -> None:
         bucket, key = src
         print(f"Downloading s3://{bucket}/{key} to {dst}")
-        s3.download_file(bucket, key, str(dst))
+
+        try:
+            s3.download_file(bucket, key, str(dst))
+        except botocore.exceptions.ClientError:
+            msg = f"Failed to download s3://{bucket}/{key}"
+            raise RuntimeError(msg)
```

Comment:
Thanks! The `print()` before here was useful but this message could be helpful in case just one of the buckets doesn't have access since we'll be downloading them all in the threadpool